packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -591,10 +592,11 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions) {
             currentSeats[
               currentSeats.findIndex((booking) => booking.startTime.toISOString() === time.toISOString())
             ]._count.attendees,
-          bookingUid:
-            currentSeats[
-              currentSeats.findIndex((booking) => booking.startTime.toISOString() === time.toISOString())
-            ].uid,
+          bookingUid: input?.bookerUserId
```

Comment:
Why do we need that?

Reply:
This is needed if a logged in user exists then it will return bookingUid for the same date time reservation. Non logged in user is another case which i will expain.